stagecraft/apps/dashboards/models/dashboard.py

```diff
@@ -250,9 +250,14 @@ def spotlightify(self, request_slug=None):
         return modules_or_tabs
 
     def serialize(self):
+        def simple_field(field):
+            return not (field.is_relation or field.one_to_one or (
```

Comment:
Example in Django is:
```python
return not field.is_relation or field.one_to_one or (field.many_to_one and field.related_model)
```

We have:
```python
return not (field.is_relation or field.one_to_one or (field.many_to_one and field.related_model))
```

I don't think that's the same, but I'd probably need to write a truth table :)